app/infrastructure/gateways/google_news_api.rb

```diff
@@ -25,21 +25,20 @@ def initialize(token)
         end
 
         def gn_api_path(topic, result_num)
-          today = Date.today
-          to = today.strftime('%Y-%m-%d')
-          from = (today - 10).strftime('%Y-%m-%d')
-
-          path = "q=#{topic}&from=#{from}&to=#{to}&pageSize=#{result_num}"
-          # path = "q=#{topic}&from=2021-10-10&to=2021-10-20&pageSize=#{result_num}"
-          #  puts path
+          path = "q=#{topic}&from=2021-10-10&to=2021-10-20&pageSize=#{result_num}"
+          # the comment below is for vcr
+          # today = Date.today
+          # to = today.strftime('%Y-%m-%d')
+          # from = (today - 10).strftime('%Y-%m-%d')
+          # path = "q=#{topic}&from=#{from}&to=#{to}&pageSize=#{result_num}"
```

Comment:
I know you want to keep it for next time if VCR changes the result. However, pushing these comment codes will make it untidy and confuse others.
In the industry, people will leave and new members will join this team, so it's really hard for new members to understand what are these comment codes for.
I would suggest you post this into [Wiki](https://github.com/SOA-nuts/GoogleNewsAPI/wiki), Wiki is the place for elaborating more about your code and how to use it for your team members.
Thus, you can copy these codes then paste them while you need them. That's the way the industry does, always make the code readable, understandable, and no useless code and comment.

Reply:
Got it.
Thanks for advice!